test_genie_python_using_simple.py

```diff
@@ -270,4 +270,80 @@ def test_GIVEN_waiting_for_value_above_low_limit_on_block_WHEN_block_reaches_lim
         set_pv_thread.start()
         g.waitfor_block(block=self.block_name, lowlimit=low_limit, maxwait=self.max_wait)
 
-        assert_that(set_pv_thread.is_alive(), is_(True), "Waitfor should have finished because block has changed to the limit")
\ No newline at end of file
+        assert_that(set_pv_thread.is_alive(), is_(True), "Waitfor should have finished because block has changed to the limit")
+
+class TestRunControl(unittest.TestCase):
+
+    def setUp(self):
+        g.set_instrument(None)
+        load_config_if_not_already_loaded(SIMPLE_CONFIG_NAME)
+        self.block_name = "FLOAT_BLOCK"
+        self.wait_before = 1
+        self.wait_after = 2
+        self.max_wait = (self.wait_before + self.wait_after) * 2
```

Comment:
Are these wait variables used?

Reply:
No they are not - copied from an earlier test